{"hierarchy":{"paths":[["doc:\/\/swift-argument-parser.ArgumentParser\/documentation\/ArgumentParser","doc:\/\/swift-argument-parser.ArgumentParser\/documentation\/ArgumentParser\/ExitCode"]]},"kind":"symbol","abstract":[{"type":"text","text":"An exit code that indicates successful completion of a command."}],"variants":[{"paths":["\/documentation\/argumentparser\/exitcode\/success"],"traits":[{"interfaceLanguage":"swift"}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"identifier":{"url":"doc:\/\/swift-argument-parser.ArgumentParser\/documentation\/ArgumentParser\/ExitCode\/success","interfaceLanguage":"swift"},"metadata":{"roleHeading":"Type Property","externalID":"s:14ArgumentParser8ExitCodeV7successACvpZ","role":"symbol","fragments":[{"text":"static","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"success"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:14ArgumentParser8ExitCodeV","kind":"typeIdentifier","text":"ExitCode"}],"title":"success","modules":[{"name":"ArgumentParser"}],"symbolKind":"property"},"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["macOS"],"languages":["swift"],"tokens":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"text":"let","kind":"keyword"},{"kind":"text","text":" "},{"text":"success","kind":"identifier"},{"text":": ","kind":"text"},{"identifier":"doc:\/\/swift-argument-parser.ArgumentParser\/documentation\/ArgumentParser\/ExitCode","kind":"typeIdentifier","preciseIdentifier":"s:14ArgumentParser8ExitCodeV","text":"ExitCode"}]}]}],"references":{"doc://swift-argument-parser.ArgumentParser/documentation/ArgumentParser/ExitCode":{"abstract":[{"text":"An error type that only includes an exit code.","type":"text"}],"type":"topic","identifier":"doc:\/\/swift-argument-parser.ArgumentParser\/documentation\/ArgumentParser\/ExitCode","title":"ExitCode","role":"symbol","url":"\/documentation\/argumentparser\/exitcode","navigatorTitle":[{"kind":"identifier","text":"ExitCode"}],"fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"text":"ExitCode","kind":"identifier"}],"kind":"symbol"},"doc://swift-argument-parser.ArgumentParser/documentation/ArgumentParser/ExitCode/success":{"fragments":[{"kind":"keyword","text":"static"},{"text":" ","kind":"text"},{"text":"let","kind":"keyword"},{"kind":"text","text":" "},{"text":"success","kind":"identifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:14ArgumentParser8ExitCodeV","text":"ExitCode"}],"kind":"symbol","abstract":[{"text":"An exit code that indicates successful completion of a command.","type":"text"}],"url":"\/documentation\/argumentparser\/exitcode\/success","title":"success","identifier":"doc:\/\/swift-argument-parser.ArgumentParser\/documentation\/ArgumentParser\/ExitCode\/success","type":"topic","role":"symbol"},"doc://swift-argument-parser.ArgumentParser/documentation/ArgumentParser":{"title":"ArgumentParser","type":"topic","abstract":[{"text":"Straightforward, type-safe argument parsing for Swift.","type":"text"}],"identifier":"doc:\/\/swift-argument-parser.ArgumentParser\/documentation\/ArgumentParser","role":"collection","url":"\/documentation\/argumentparser","kind":"symbol"}}}